src/main/java/de/rabea/gui/BoardView.java

```diff
@@ -2,40 +2,47 @@
 
 import de.rabea.game.Board;
 import de.rabea.game.Mark;
-import javafx.geometry.Pos;
 import javafx.scene.Parent;
-import javafx.scene.control.Label;
 import javafx.scene.layout.GridPane;
+import javafx.scene.text.Text;
 
 import java.util.Map;
 
-import static de.rabea.game.Mark.*;
+import static de.rabea.game.Mark.EMPTY;
+import static javafx.geometry.Pos.CENTER;
 
 public class BoardView {
 
-    private final ClickHandler clickHandler;
+    private final ClickHandler emptyCellClickHandler;
+    private final ClickHandler fullCellClickHandler;
 
-    public BoardView(ClickHandler clickHandler) {
-        this.clickHandler = clickHandler;
+    public BoardView(ClickHandler emptyCellClickHandler, ClickHandler fullCellClickHandler) {
+        this.emptyCellClickHandler = emptyCellClickHandler;
+        this.fullCellClickHandler = fullCellClickHandler;
     }
 
-    public Parent draw(Board board) {
+    public Parent draw(Board board, boolean positionInUse) {
         GridPane gridPane = new GridPane();
-        gridPane.setAlignment(Pos.CENTER);
+        gridPane.setAlignment(CENTER);
         for (Map.Entry<Integer, Mark> entry : board.cellsWithIndex().entrySet()) {
             int position = entry.getKey();
             Mark cell = entry.getValue();
             int column = position % board.getDimension();
             int row = position / board.getDimension();
             if (cell == EMPTY) {
-                JavaFXButton javaFXButton = new JavaFXButton(clickHandler, (position + 1) + "", position + "", "cell");
-                gridPane.add(javaFXButton.getActualButton(), column, row);
+                JavaFXButton activeButton = new JavaFXButton(emptyCellClickHandler, "_", position + "", "active-button");
+                gridPane.add(activeButton.getActualButton(), column, row);
             } else {
```

Comment:
@felipesere I think the code is easier to read if I use a temporary variable here. What do you think?